backend/guides/code.md

```diff
@@ -0,0 +1,159 @@
+# Coding
+
+## Source code
+
+### Naming above all
+
+The source code you write is intended for only two audiences
+
+- other people,
+- compiler/interpreter.
+
+The compiler doesn't care about the naming, doesn't have to understand the semantics, therefore the readability and understandability are the most important aspects above all.
+
+It's exactly like when two people from different countries trying to express themselves with language the other doesn't understand. Or even worse, they speak the same language and each of them are using different naming for the same things.
+
+When you call something 'banana', it should be banana and not an orange.
+
+### Learn to love old code
```

Comment:
I believe that Boy Scout Rule is a better way: https://deviq.com/boy-scout-rule/ - making small enhancements to the code along the way. So up for a discussion?

Reply:
Generally I agree. So we replace?

Reply:
I think the Boy Scout Rule generally describes this well and I think you can replace it with the reference with the quote :+1:

Reply:
Updated, thanks